hmy/api_backend.go

```diff
@@ -587,6 +587,18 @@ func (b *APIBackend) GetSuperCommittees() (*quorum.Transition, error) {
 		quorum.NewRegistry(stakedSlotsThen),
 		quorum.NewRegistry(stakedSlotsNow)
 
+	then.MedianStake, then.SlotPurchases =
+		committee.ComputeMedianRawStakeForShardCommittee(
+			b.hmy.BlockChain(),
+			prevCommittee.StakedValidators().Addrs,
+		)
+
+	now.MedianStake, now.SlotPurchases =
+		committee.ComputeMedianRawStakeForShardCommittee(
```

Comment:
Actually no need to be this complicated. Just add a field in the final result:

 {
                            "is-harmony-slot": false,
                            "earning-account": "one1x8kgyyqgslzrm7jrlj4m7rpg285etdywjk7gez",
                            "bls-public-key": "5e8635085788492a87d3eab69732ae72a55087e95c210ad20a726dd169a69b5792cb94616f6eccc4ea0a54a4a77ca88c",
                            "voting-power-unnormalized": "0.026785021562679984",
                            "voting-power-%": "0.008571206900057595",
                            "effective-stake": "5920773850000000000000000.000000000000000000"
                            "raw-stake": NEW_FIELD,
                        },

The raw-stake is just validatorSnap.TotalStake / size(validatorSnap.BLSKeys). 

No need to add new func like: ComputeMedianRawStakeForShardCommittee. Just loop through all the validators in the decider result and for each validator address, fetch the snapshot and compute the raw-stake and put it in the result.